modules/transactions.js

```diff
@@ -547,6 +547,11 @@ shared.addTransactions = function (req, cb) {
 			}
 		}
 
+		var mainNetGeneratorPublicKey = 'd121d3abf5425fdc0f161d9ddb32f89b7750b4bdb0bff7d18b191d4b4bafa6d4';
```

Comment:
So `mainnetGenesisPublicKey` :D Or maybe add that to `helpers/exceptions.js` or separate file?